### ISSUE TITLE
Does Predefined Concept Data provided in 'csi300_stock2concept.npy' contain future information?

### ISSUE THREAD

**Issue by xmm1016** [Wentao-Xu/HIST#11]
Dear authors,

The paper mentioned that the change of stocks’ industry and businesses causes the number of predefined concepts change dynamically. However, it seems like that the same predefined concept data was used in training and testing process.   if stock2concept_matrix is created using data from 01/01/2007 to 12/31/2020, with training set (from 01/01/2007 to 12/31/2014), validation set (from 01/01/2015 to 12/31/2016), and test set (from 01/01/2017 to 12/31/2020), there is a problem that the training process is influenced by future information from stock2concept_matrix. 
I would appreciate that you could provide the extraction process of the stock-to-predefined concept data.

**Comment by Wentao-Xu** [comment]
Hi, the csi300_stock2concept.npy do not contain any future information, especially the information in the test set.
